storybook/.storybook/preview.tsx

```diff
@@ -4,8 +4,7 @@ import { MainContent, MuiThemeProvider } from "@comet/admin";
 import { DateFnsLocaleProvider } from "@comet/admin-date-time";
 import { createCometTheme } from "@comet/admin-theme";
 import { createTheme as createMuiTheme, GlobalStyles } from "@mui/material";
-import { select, withKnobs } from "@storybook/addon-knobs";
-import { addDecorator, addParameters } from "@storybook/react";
+import {  addParameters, ArgTypes, DecoratorFn } from "@storybook/react";
```

Comment:
```suggestion
import { addParameters, ArgTypes, DecoratorFn } from "@storybook/react";
```

Reply:
I added a lint for the .storybook/ folder to fix this: 27282a59ed0a825959ce261989cc93a8ec726bab